### DISCUSSION TITLE
Half screen modal view for iOS

### DISCUSSION THREAD

**Discussion by andrrrr** [MvvmCross/MvvmCross#4096] [Q&A]
Hi there, 
I am trying to recreate a simple modal view popup that covers only half of the screen (or depends on the content height), so far I'm able to show a normal MvxViewController with MvxModalPresentation attribute like this:
   ```
 [MvxModalPresentation(
     ModalPresentationStyle = UIModalPresentationStyle.FormSheet,
     WrapInNavigationController = true
 )]
public partial class LogoutViewController : BaseViewController<LogoutViewModel> ...
```
however it covers almost full screen no matter what I try. There is an attribute PreferredContentSize, but MvvmCross documentation states that it works on iPad only. Is there a way to do what I'm trying to do?

**Comment by Cheesebaron** [comment]
Not on iPhone no, that is not how modals work.

**Reply by andrrrr** [reply]
ok I understand it's not going to be out-of-the-box, but is there a different way to show a view half screen on top of another view?

**Reply by Cheesebaron** [reply]
This is not really a MvvmCross concern.

**Reply by andrrrr** [reply]
what a legend, thanks for help

**Reply by Cheesebaron** [reply]
Like it or not. I am not going to answer arbitrary unrelated questions here. Use StackOverflow instead.
